src/main/kotlin/com/kryszak/gwatlin/http/BaseHttpClient.kt

```diff
@@ -51,11 +51,14 @@ internal open class BaseHttpClient(
         return processResult(result, ErrorResponse(response, RetrieveError::class.java))
     }
 
-    protected fun addDefaultHeaders(request: Request, language: String?) {
+    protected fun addDefaultHeaders(request: Request, language: ApiLanguage?) {
         schemaVersion?.let { request.appendHeader("X-Schema-Version" to it) }
-        request.appendHeader(Headers.ACCEPT_LANGUAGE to (language ?: defaultLanguage))
+
+        // Only send the Accept-Language header when a specific language has been requested
```

Comment:
I think this comment doesn't contribute much, we don't have corresponding comment for schema version header